### ISSUE TITLE
"mv /var/lib/mysql/mysql => /var/lib/mysql/mysql.tmp" also result in the deleting of /var/lib/mysql data:

### ISSUE THREAD

**Issue by indicolite** [codership/galera#463]
below steps also result in the deleting of /var/lib/mysql data:
```
[root@bingo mysql]# pwd
/var/lib/mysql
[root@syluo mysql]# ls
aria_log.00000001  galera.cache        gvwstate.dat  ib_logfile1        mysql.sock          tc.log
aria_log_control   gcache.page.000006  ibdata1       multi-master.info  mysql_upgrade_info  test
fitos_sso          grastate.dat        ib_logfile0   mysql              performance_schema
[root@syluo mysql]# mv mysql mysql.temp
```
then restart the node,
still stopped,
acutally, /var/lib/mysql/* is empty.

```
Server version: 10.1.12-MariaDB MariaDB Server

Copyright (c) 2000, 2016, Oracle, MariaDB Corporation Ab and others.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

MariaDB [(none)]> show status like '%wsrep%';
+------------------------------+----------------------------------------------+
| Variable_name                | Value                                        |
+------------------------------+----------------------------------------------+
| wsrep_apply_oooe             | 0.000000                                     |
| wsrep_apply_oool             | 0.000000                                     |
| wsrep_apply_window           | 0.000000                                     |
| wsrep_causal_reads           | 0                                            |
| wsrep_cert_deps_distance     | 0.000000                                     |
| wsrep_cert_index_size        | 0                                            |
| wsrep_cert_interval          | 0.000000                                     |
| wsrep_cluster_conf_id        | 1                                            |
| wsrep_cluster_size           | 1                                            |
| wsrep_cluster_state_uuid     | cbfa2e0f-3486-11e7-b2d1-b3288aae4c06         |
| wsrep_cluster_status         | Primary                                      |
| wsrep_commit_oooe            | 0.000000                                     |
| wsrep_commit_oool            | 0.000000                                     |
| wsrep_commit_window          | 0.000000                                     |
| wsrep_connected              | ON                                           |
| wsrep_evs_delayed            |                                              |
| wsrep_evs_evict_list         |                                              |
| wsrep_evs_repl_latency       | 1.137e-06/4.2208e-06/8.371e-06/2.80406e-06/5 |
| wsrep_evs_state              | OPERATIONAL                                  |
| wsrep_flow_control_paused    | 0.000000                                     |
| wsrep_flow_control_paused_ns | 0                                            |
| wsrep_flow_control_recv      | 0                                            |
| wsrep_flow_control_sent      | 0                                            |
| wsrep_gcomm_uuid             | a7d8359d-3486-11e7-811e-13d29a7b53b4         |
| wsrep_incoming_addresses     | 192.168.101.141:3306                         |
| wsrep_last_committed         | 0                                            |
| wsrep_local_bf_aborts        | 0                                            |
| wsrep_local_cached_downto    | 18446744073709551615                         |
| wsrep_local_cert_failures    | 0                                            |
| wsrep_local_commits          | 0                                            |
| wsrep_local_index            | 0                                            |
| wsrep_local_recv_queue       | 0                                            |
| wsrep_local_recv_queue_avg   | 0.500000                                     |
| wsrep_local_recv_queue_max   | 2                                            |
| wsrep_local_recv_queue_min   | 0                                            |
| wsrep_local_replays          | 0                                            |
| wsrep_local_send_queue       | 0                                            |
| wsrep_local_send_queue_avg   | 0.000000                                     |
| wsrep_local_send_queue_max   | 1                                            |
| wsrep_local_send_queue_min   | 0                                            |
| wsrep_local_state            | 4                                            |
| wsrep_local_state_comment    | Synced                                       |
| wsrep_local_state_uuid       | cbfa2e0f-3486-11e7-b2d1-b3288aae4c06         |
| wsrep_protocol_version       | 7                                            |
| wsrep_provider_name          | Galera                                       |
| wsrep_provider_vendor        | Codership Oy <info@codership.com>            |
| wsrep_provider_version       | 3.12(r9921e73)                               |
| wsrep_ready                  | ON                                           |
| wsrep_received               | 2                                            |
| wsrep_received_bytes         | 144                                          |
| wsrep_repl_data_bytes        | 0                                            |
| wsrep_repl_keys              | 0                                            |
| wsrep_repl_keys_bytes        | 0                                            |
| wsrep_repl_other_bytes       | 0                                            |
| wsrep_replicated             | 0                                            |
| wsrep_replicated_bytes       | 0                                            |
| wsrep_thread_count           | 33                                           |
+------------------------------+----------------------------------------------+
57 rows in set (0.09 sec)
```

**Comment by indicolite** [comment]
see: https://github.com/codership/galera/issues/462
fixed with mariadb-10.1.12.

**Comment by indicolite** [comment]
Close this bug now.
